src/components/Link.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div @click="$router.push(`${link}`).catch((error) => routerError(error))">
+    <router-link :to="`${link}`" :class="getClass()">
+      {{ name }}
+    </router-link>
+  </div>
+</template>
+
+<script>
+export const ViewType = {
+  Button: 1, //Display as dashboard button
+  MenuItem: 2, // Display as menu item
```

Comment:
and camelCase?

Reply:
This was something Seena added as part of the styling. I think it is like that because it is an enum.